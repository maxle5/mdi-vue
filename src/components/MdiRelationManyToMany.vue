
<template>
  <svg :width="size" :height="size" :viewBox="viewBox">
      <path d="M22 13V19H21L19 17H11V9H5L3 11H2V5H3L5 7H13V15H19L21 13Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiRelationManyToMany",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
</script>
