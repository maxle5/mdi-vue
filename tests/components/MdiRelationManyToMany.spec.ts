
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiRelationManyToMany from "../../src/components/MdiRelationManyToMany.vue";

test("MdiRelationManyToMany snapshot", () => {
  const wrapper = mount(MdiRelationManyToMany, {});
  expect(wrapper.html()).toMatchSnapshot();
});
