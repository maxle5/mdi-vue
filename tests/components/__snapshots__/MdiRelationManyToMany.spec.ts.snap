// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiRelationManyToMany snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M22 13V19H21L19 17H11V9H5L3 11H2V5H3L5 7H13V15H19L21 13Z" style="fill: currentColor;"></path>
</svg>"
`;
